guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -1,55 +1,267 @@
+| Unknown execution status | OK | 2 or 3
+| Last execution cancelled | OK | 4
+|===
 
-The *Build* sub-status has two possible values {endash} *pending* and *built* that are both mapped to global *OK* value.
+ifndef::foreman-deb[]
+Compliance::
+Indicates if the host is compliant with OpenSCAP policies.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Compliant | OK | 0
+| Inconclusive | Warning | 1
+| At least one incompliant | Error | 2
+|===
+endif::[]
 
-The *Configuration* status has more possible values that map to the global status as follows:
+OVAL scan::
+Indicates if there are any vulnerabilities found on the host
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| No vulnerabilities found | OK | 0
+| Vulnerabilities found | Warning | 1
+| Vulnerabilities with available patch found | Error | 2
+|===
 
-.Sub-statuses that map to the global *OK* status
+ifdef::satellite,orcharhino,katello[]
+Inventory::
+Indicates if the host is synchronized to {RHCloud}.
+{ProjectServer} performs the synchronization itself but only uploads basic information to {RHCloud}.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Host was not uploaded to your RH cloud inventory | Warning | 0
+| Successfully uploaded to your RH cloud inventory | OK | 1
+|===
 
-Active::
-During the last run, some resources were applied.
+Insights::
+Indicates if the host is synchronized to {RHCloud}.
+This synchronization is performend by the host.
+The host uploads more information than the {ProjectServer}.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Reporting | OK | 0
+| Not reporting | Error | 1
+|===
 
-Pending::
-During the last run, some resources would be applied but your configuration management integration was configured to run in `noop` mode.
+RHEL Lifecycle::
+Indicates the current state of the {RHEL} operating system installed on the host.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Unknown | OK | 0
+| Full support | OK | 1
+| Maintenance support | OK | 2
+| Approaching end of maintenance support | Warning | 3
+| Extended support | OK | 4
+| Approaching end of support | Warning | 5
+| Support ended | Error | 6
+|===
 
-No changes::
-During the last run, nothing changed.
+Errata::
+Indicates if Errata is available on the host.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Up to date | OK | 0
+| Unknown | Warning | 1
+| Needed errata | Error | 2
+| Needed security errata | Error | 3
+|===
 
-No reports::
-This can be both a *Warning* or *OK* sub-status.
-This occurs when there are no reports but the host uses, for example, an associated configuration management proxy or `always_show_configuration_status` setting is set to `true`, it maps to *Warning*.
+Subscription::
```

Comment:
RHEL subscription or Foreman/candlepin subscription?

Reply:
Red-hat subscription acquired through subscription-manager. Should I make it satellite-only?

Reply:
No, then it'll also affect Foreman/orcharhino. Instead, I would reword this to "RHEL subscription" if possible.

Reply:
I would assume "no" because you don't need a valid subscription for Debian. But you can manage RHEL hosts using Foreman/orcharhino.

I have checked on Foreman 3.9 with SCA off for AlmaLinux 8:

![Screenshot from 2024-02-21 16-04-23](https://github.com/theforeman/foreman-documentation/assets/12595287/b3613014-1c71-4895-a490-e1e6258a81d6)
 
with SCA on, it's green by default (Debian 12):

![image](https://github.com/theforeman/foreman-documentation/assets/12595287/87d477fe-ca7c-4e76-a76a-e28315ffe347)

Reply:
I guess I should keep it visible for foreman-deb if it's present.

Reply:
If should only be visible for katello,satellite,orcharhino because you can only do content management with Katello plug-in. However, the "Subscription" status refers to the client OS and appears to be valid for all registered hosts (not just RHEL) AFAIK.

Reply:
So leaving visibility only for katello,satellite,orcharhino.